bokeh/models/annotations.py

```diff
@@ -1096,6 +1097,44 @@ class Whisker(Annotation):
 
     level = Override(default="underlay")
 
+class InfoPane(Annotation):
+    ''' Renders a info-pane.
+
+    '''
+    level = Override(default="overlay")
+
+    # TODO: can be modified to be anchored at corners as well
+    anchor = Enum(TooltipAttachment, help="""
+    Whether the pane should be anchored at the left or right of the pane
+    or above or below it, or if it should be automatically placed
+    in the horizontal or vertical dimension.
+    """)
+
+    x = Either(Auto, NumberSpec(), default=0, help="""
+    Initial value of the anchor position for pane to apply to the x-coordinate.
+    """)
+
+    y = Either(Auto, NumberSpec(), default=0, help="""
+    Initial value of the anchor position for pane to apply to the y-coordinate.
+    """)
+
+    anchor_units = Enum(SpatialUnits, default='data', help="""
```

Comment:
This should be something like `position_units` (or at least, not `anchor_units` since the anchor is a named location of the pane itself)

Reply:
I will change this.